Library/Homebrew/load_path.rb

```diff
@@ -7,3 +7,5 @@
 $LOAD_PATH.push(HOMEBREW_LIBRARY_PATH.to_s) unless $LOAD_PATH.include?(HOMEBREW_LIBRARY_PATH.to_s)
 
 require "vendor/bundle/bundler/setup"
+
+$LOAD_PATH.uniq!
```

Comment:
A possible extension I was thinking about could be to also trim any paths that don't `:exist?` or `:directory?`. Might be overkill, though.

Reply:
That would trim 15 paths on my system, so it might be worth it. Notably it would trim the `universal-darwin-19` paths in setup.rb (they are Catalina specific paths) and Ruby's own stuff like `site_ruby`.

Reply:
Those paths should actually be `x86_64-darwin13` even on Catalina because it depends on the system Ruby was built on but whatever.